bitseq/sequence.go

```diff
@@ -262,19 +269,11 @@ func (h *Handle) runConsistencyCheck() bool {
 // It looks for a corruption signature that may happen in docker 1.9.0 and 1.9.1.
 func (h *Handle) CheckConsistency() error {
 	for {
-		h.Lock()
-		store := h.store
-		h.Unlock()
-
-		if store != nil {
-			if err := store.GetObject(datastore.Key(h.Key()...), h); err != nil && err != datastore.ErrKeyNotFound {
-				return err
-			}
+		if err := h.Refresh(); err != nil {
```

Comment:
`h.store` is set at handle creation, and will not be changed after that. I wanted to avoid a lock if not strictly necessary,although I know we lock around access to `h.store` in other places.